pkg/sdk/random_test.go

```diff
@@ -14,6 +15,12 @@ var (
 	emptyDatabaseObjectIdentifier            = NewDatabaseObjectIdentifier("", "")
 	emptySchemaObjectIdentifier              = NewSchemaObjectIdentifier("", "", "")
 	emptySchemaObjectIdentifierWithArguments = NewSchemaObjectIdentifierWithArguments("", "", "")
+
+	// TODO [SNOW-1843440]: create using constructors (when we add them)?
+	dataTypeNumber, _  = datatypes.ParseDataType("NUMBER(36, 2)")
```

Comment:
Maybe they should be created in `func init()` to ensure they don't throw the error?

Reply:
I will add this as a TODO